controllers/designate_controller.go

```diff
@@ -552,10 +556,12 @@ func (r *DesignateReconciler) reconcileNormal(ctx context.Context, instance *des
 			condition.SeverityWarning,
 			condition.ServiceConfigReadyErrorMessage,
 			err.Error()))
+		r.Log.Info("createHashOfInputHashes failed", err)
```

Comment:
This will cause a panic because the `err` variable is unexpected here.  You would have to first create a string using something like `fmt.Sprintf("some-text: %v", err)`.